layers/park/park.yaml

```
@@ -1,23 +1,27 @@
 layer:
   id: "park"
   description: |
-      The park layer contains parks from OpenStreetMap tagged with
-      [`boundary=national_park`](http://wiki.openstreetmap.org/wiki/Tag:boundary%3Dnational_park),
-      [`boundary=protected_area`](http://wiki.openstreetmap.org/wiki/Tag:boundary%3Dprotected_area),
-      or [`leisure=nature_reserve`](http://wiki.openstreetmap.org/wiki/Tag:leisure%3Dnature_reserve).
+    The park layer in OpenMapTiles contains natural and protected areas from OpenStreetMap, 
+    such as parks tagged with [`boundary=national_park`](https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dnational_park),
+    [`boundary=protected_area`](https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dprotected_area),
+    or [`leisure=nature_reserve`](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dnature_reserve).
+    This layer also includes boundaries for indigenous lands tagged with 
+    [`boundary=aboriginal_lands`](https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dprotected_area).
+    While indigenous boundaries are not synonymous with parks, they are included in this layer for technical reasons related to data processing.
```

They aren’t even similar to parks:

```suggestion
    Indigenous boundaries are not parks, but they are included in this layer for technical reasons related to data processing.
```